### PR TITLE
@TT-4: Implementing govwales typography

### PR DESCRIPTION
**Pull request by adamccfc**

### CODE REVIEW

File: app/assets/stylesheets/govwales/base/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+// Font stacks
+$times-new-roman: "Times New Roman", Times, Georgia, "DejaVu Serif", serif;
+$times:           Times, "Times New Roman", Georgia, "DejaVu Serif", serif;
+$georgia:         Georgia, "Times New Roman", "DejaVu Serif", serif;
+
+$verdana:         Verdana, Tahoma, "DejaVu Sans", sans-serif;
+$tahoma:          Tahoma, Verdana, "DejaVu Sans", sans-serif;
+$helvetica:       Helvetica, Arial, "Nimbus Sans L", sans-serif;
+$arial:           Arial, Helvetica, "Nimbus Sans L", sans-serif;
+$courier:         "Courier New", "DejaVu Sans Mono", monospace, sans-serif;
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/base/_typography.scss]: Remove tab spacing.

File: app/assets/stylesheets/govwales/base/_typography.scss [hunk 2]
```diff
@@ -0,0 +1,116 @@
+// =======================================================================
+// BASE - CSS TYPOGRAPHIC VARIABLES
+// This file handles the base typography variables.
+// =======================================================================
+
+// Fontkit
+$rockwell-bold:"Rockwell W01 Bold", Arial, Helvetica, sans-serif;
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/base/_typography.scss]: space after :
& line 8

File: app/assets/stylesheets/govwales/base/_typography.scss [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+$label-font-size: 18px;
+$label-line-size: 24px;
+
+// P, ol, ul Desktop font-sizes
+$label-d-font-size: 20px;
+$label-d-line-size: 26px;
+
+// Hard kerning variables
+$wide_spacing:0.08em; // Wide letter spacing (Uppercase headings)
+$tight_spacing:-0.05em; // Tight letter spacing (H1 - H6 normal headings)
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/base/_typography.scss]: space after :
Line 80 & 81

File: app/assets/stylesheets/govwales/base/_typography.scss [hunk 4]
```diff
@@ -0,0 +1,116 @@
+// =======================================================================
+// BASE - CSS TYPOGRAPHIC VARIABLES
+// This file handles the base typography variables.
+// =======================================================================
+
+// Fontkit
+$rockwell-bold:"Rockwell W01 Bold", Arial, Helvetica, sans-serif;
+$rockwell-bolditalic:"RockwellW01-BoldItalic", Arial, Helvetica, sans-serif;
```
Comment (markheathy) [inline app/assets/stylesheets/govwales/base/_typography.scss]: Do we want to remove any of these or do it at a later date @adamccfc ?

**Review by markheathy — CHANGES_REQUESTED** [review]
Small adjustments @adamccfc, thanks.

File: app/assets/stylesheets/govwales/base/_typography.scss (hunk 4 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/base/_typography.scss]: Do it later @markheathy - I'm copying files across from the Drupal project. We can clean it up once MVP version is done.

File: app/assets/stylesheets/govwales/base/_typography.scss (hunk 2 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/base/_typography.scss]: This is done

File: app/assets/stylesheets/govwales/base/_typography.scss (hunk 1 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/base/_typography.scss]: Removed

File: app/assets/stylesheets/govwales/base/_typography.scss (hunk 3 above)
Comment (adamccfc) [reply app/assets/stylesheets/govwales/base/_typography.scss]: Done

File: app/assets/stylesheets/govwales/base/_typography.scss (hunk 4 above)
Comment (markheathy) [reply app/assets/stylesheets/govwales/base/_typography.scss]: No worries

**Review by markheathy — APPROVED** [review]
Approved for merge @adamccfc .

**Pull request merged**